libs/langchain-mongodb/langchain_mongodb/chat_message_histories.py

```diff
@@ -112,7 +114,12 @@ def __init__(
             self.client = client
         elif connection_string:
             try:
-                self.client = MongoClient(connection_string)
+                self.client = MongoClient(
+                    connection_string,
+                    driver=DriverInfo(
+                        name="Langchain", version=version("langchain-mongodb")
```

Comment:
Can we overload the version info to also include info about the specific feature? 
for instance here it would be version=version("langchain-mongodb-chat-message")

Reply:
Version is only tracked at the package level.  If we want to include more info, we usually add it to the name, like `PyMongo|c|async`, so in this case it would be `Langchain|chat-message`.